lib/logster/logger.rb

```diff
@@ -4,29 +4,25 @@
 
 module Logster
   class Logger < ::Logger
-    LOGSTER_ENV = "logster_env".freeze
+    LOGSTER_ENV = "logster_env"
 
     attr_accessor :store, :skip_store
     attr_reader :chained
 
     def initialize(store)
       super(nil)
       @store = store
-      @override_levels = nil
       @chained = []
       @skip_store = false
+      @logster_override_level_key = "logster_override_level_#{object_id}"
```

Comment:
For best performance, it would be better to `.to_sym` at the end.
In general methods accepting both Symbols and Strings are faster with Symbols.
One reason in this case is the Ruby implementation most likely converts to a Symbol internally before anything (but then on every access vs once), to ensure this works:
```ruby
irb(main):001:0> Thread.current["abc"] = 1
=> 1
irb(main):002:0> Thread.current[:"abc"]
=> 1
```